ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.h

```diff
@@ -66,10 +66,12 @@ ORK1_CLASS_AVAILABLE
 /// The `scriptMessageHandler` is initialized as `nil`. Any incoming script messages received while `scriptMessageHandler` is nil will be enqueued for later processing. Interactivity can thus be delayed until this view controller is ready to display, for example by waiting to set a `scriptMessageHandler` until this view controller's `viewWillAppear` lifecycle event.
 /// - Parameters:
 ///   - step: Must be an `ORK1WebViewStep`.
+///   - result: The previous step result for this step, if any. The view controller will use this as the source of its `result` value until any user interaction updates the result.
 ///   - scriptMessageNames: A set of custom WebKit message names to declare as supported. See ``scriptMessageHandler``.
 ///   - remoteURLCachePolicy: Cache policy for loading URL-based web view steps.
 ///   - remoteURLTimeoutInterval: Timeout interval for loading URL-based web view steps.
 - (instancetype)initWithStep:(ORK1Step *)step
+                      result:(nullable ORK1Result *)result
```

Comment:
Is there any reason this is an `ORK1Result` and not an `ORK1StepResult`?

Reply:
Only because ORK1StepViewController's [initWithStep:result:](https://github.com/CareEvolution/ResearchKit/blob/32dc55f43c0170c924ecbc69e6e05854d48a9f80/ORK1Kit/ORK1Kit/Common/ORK1StepViewController.h#L203) is declared using ORK1Result